packages/amplication-server/prisma/schema.prisma

```diff
@@ -28,6 +28,29 @@ model Workspace {
   name      String   @unique(map: "Workspace.name_unique")
   apps      App[]
   users     User[]
+  gitOrganizations GitOrganization[]
+}
+
+model GitOrganization {
+  id             String          @id @default(cuid())
+  provider       EnumSourceControlService
+  name           String
+  installationId Int
```

Comment:
is that really an integer? I think it is safer to use a string to be able to work with different providers